src/project-tests/technical-docs-tests.js

```diff
@@ -349,35 +349,36 @@ export default function createTechnicalDocsPageTests() {
       function() {
         const windowWidth =
           document.documentElement.clientWidth || window.innerWidth;
+        const windowHeight =
+          document.documentElement.clientHeight || window.innerHeight;
         assert.isAbove(
           windowWidth,
           850,
           'Please run this test in a larger window (before any media queries) '
         );
         var navbar = document.getElementById('navbar');
-        assert.approximately(
+        // navbar should be in top-left quadrant
+        // navbar on left side
+        assert.isBelow(
           navbar.getBoundingClientRect().left,
-          0,
-          10,
+          (windowWidth / 2) - navbar.getBoundingClientRect().width,
           'Left of bounding rectangle is not correct.'
```

Comment:
I know this isn't your change, but while you are here could you make this message clearer? I don't think it makes sense to Campers. What's a bouding rectangle? I think it should be something more like "The element with id="navbar" is not on the left side of the screen". What do you think?